public/docs/advanced/commands.md

```diff
@@ -1,129 +1,215 @@
+Interactively debug any transaction on the blockchain.
 
-```none
-$ truffle create migration MyContract
+```shell
+truffle debug <transaction_hash>
 ```
 
-### create test
+Will start an interactive debugging session on a particular transaction. Allows you to step through each action and replay. See the [Debugging your contracts](/docs/getting_started/debugging) section for more details.
+
+
+Parameters:
+
+* `<transaction_hash>`: Transaction ID to use for debugging.
+
+
+### deploy
+
+Alias for `migrate`. See `migrate` for details.
+
+
+### develop
 
-Helper method to scaffold a new test for a contract. Name must be camel-case.
+Open a console with a local TestRPC
 
-```none
-$ truffle create test MyTest
+```shell
+truffle develop
 ```
 
+Spawns a local development blockchain, and allows you to interact with contracts via the command line. Additionally, many Truffle commands are additionally available within the console (without the `truffle` prefix).
+
+If you want an interactive console but want to use an existing blockchain, use `truffle console`.
+
+See the [Using the console](/docs/getting_started/console) section for more details.
+
+
 ### exec
 
-Execute a Javascript file within the Truffle environment. This will include `web3`, set the default provider based on the network specified (if any), and include your contracts as global objects while executing the script. Your script must export a function that Truffle can run. See the [Writing external scripts](/docs/getting_started/scripts) section for more details.
+Execute a JS module within this Truffle environment.
 
-```none
-$ truffle exec /path/to/my/script.js
+```shell
+truffle exec <script.js> [--network <name>] 
 ```
 
-Optional parameter:
+This will include `web3`, set the default provider based on the network specified (if any), and include your contracts as global objects while executing the script. Your script must export a function that Truffle can run. 
+
+See the [Writing external scripts](/docs/getting_started/scripts) section for more details.
+
+Parameters:
+
+* `<script.js>` JavaScript file to be executed. Can include path information if the script does not exist in the current directory.
+
+Optional parameters:
+
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
+
+
+### help
+
+Display a list of all commands and then exit.
+
+```shell
+truffle help
+```
 
-* `--network name`: Specify the network to use, using artifacts specific to that network.
 
 ### init
 
-Create a completely new app within the current working directory. Will add default contracts, tests and frontend configuration.
+Initialize new Ethereum project with example contracts and tests.
 
-```none
-$ truffle init
+```shell
+truffle init
 ```
 
+Creates a new and empty Truffle project within the current working directory. Will add default contracts, tests and frontend configuration. Takes no arguments.
```

Comment:
This no longer creates default contracts, tests and frontend configuration. So that should likely be removed. Also this sentence doesn't match the `empty Truffle project` mentioned in the previous sentence. 

This leaves the sentence: 
> Creates a new and empty Truffle project within the current working directory

This is a good summary sentence, but you added another above. I'd choose one. (I like the one on this line, your call).

Reply:
The top line of all of these entries match the blurb that shows up on the `help` output.  But good point about what `init` does. I updated the blurb in the truffle-core project (and here).